website_event_compassion/__manifest__.py

```diff
@@ -29,7 +29,7 @@
 # pylint: disable=C8101
 {
     "name": "Compassion Events Website",
-    "version": "12.0.1.0.2",
+    "version": "12.0.1.0.3",
```

Comment:
The version is new but there is no _pre_ or _post-migration_ script, is that normal ?

Reply:
Initially I wanted to put the migration here, but eventually I put it in `partner_compassion`. However it's not bad to increment the module version when we make changes.